data/reusables/github-actions/self-hosted-runner-removing-a-runner.md

```diff
@@ -1,4 +1,4 @@
-1. Under **Self-hosted runners**, locate the runner in the list. If your runner is in a group, click {% octicon "chevron-down" aria-label="The downwards chevron" %} to expand the list.
+1. Under {% if currentVersion == "free-pro-team@latest" %}_Runners_{% else %}_Self-hosted runners_{% endif %}, locate the runner in the list. If your runner is in a group, click {% octicon "chevron-down" aria-label="The downwards chevron" %} to expand the list.
```

Comment:
I think we also need to use the new `settings-sidebar-actions-runners` reusable in the "Removing..." procedures or the new step to click "Runners" is missing.

Reply:
Fixed, thank you!